README.md

```diff
@@ -169,7 +169,7 @@ The service currently supports the following Tax Regimes:
  }
 ```
 
-### Single-Error Response
```

Comment:
Is this an accidental change due to a find and replace?

There are also many other examples like this in the PR that seem like accidental changes. I noticed multiple inside logging messages where the word "Error" was used but it's not a reference to our model. I've opted for one comment here rather than repeat the comment in all the individual places. Can you have a look through the PR and change it where appropriate.

(does the change need to be made _anywhere_?)